src/NuGet.Core/NuGet.Packaging/Strings.resx

```diff
@@ -850,4 +850,26 @@ Valid from:</comment>
     <value>The {0}'s timestamping certificate is not trusted by the trust provider.</value>
     <comment>0 -Signature friendly name</comment>
   </data>
+  <data name="InvalidP7bFileExtension" xml:space="preserve">
+    <value>The file at '{0}' is invalid.  The file must be a valid PKCS #7 file with a '.p7b' extension.</value>
```

Comment:
For my own education, when do we quote a value or not?  Other messages in this PR mention a location on disk but don't quote it.  In my own PRs I'm not sure when to quote stuff.

Reply:
I would say: quote when something needs to be written as is. for example, a file path or a keyword.

Reply:
Removed this string resource.